### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kubilaykiymaci**
has been added alphanumeric support's text

### CODE REVIEW

**Comment by taflanidi** [comment]
Hey @kubilaykiymaci 
Thanks for your efforts.

I'd like to hear the motivation behind your pull request.

**Comment by kubilaykiymaci** [comment]
Hey,

I needed to use alphanumeric chars in my project and i saw this mask type in cookbook(iban types) after a long searching in this library. So i want to help to people find it easily.

File: README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
-### Important: Migration Guide: v.5
-
-We've added a new concept of cursor gravity to our library internals. A simple flag had allowed us to get rid of some logic flaws and to finalise the right-to-left functionality at the cost of backward compatibility loss.
-
+<img src="https://raw.githubusercontent.com/RedMadRobot/input-mask-ios/assets/Assets/phone_input.gif" alt="Input Mask" width="640" />
+
+### Important: Migration Guide: v.5
+
+We've added a new concept of cursor gravity to our library internals. A simple flag had allowed us to get rid of some logic flaws and to finalise the right-to-left functionality at the cost of backward compatibility loss.
+
```
Comment (kubilaykiymaci) [inline README.md]: i didn't understand showing these lines different.

Comment (taflanidi) [reply README.md]: Might be a glitch. Not a big deal.

**Comment by taflanidi** [comment]
Understood.

How about instead of just plain «alphanumeric» you add a United Kingdom IBAN mask sample? Like this:

> …
> 9. MM/YY: `[00]{/}[00]`
> 10. UK IBAN: `GB[00] [____] [0000] [0000] [0000] [00]`
> …

Other items in the same list demonstrate real life examples, maybe let's keep this consistent?

**Comment by kubilaykiymaci** [comment]
> Understood.
> 
> How about instead of just plain «alphanumeric» you add a United Kingdom IBAN mask sample? Like this:
> 
> > …
> > 9. MM/YY: `[00]{/}[00]`
> > 10. UK IBAN: `GB[00] [____] [0000] [0000] [0000] [00]`
> > …
> 
> Other items in the same list demonstrate real life examples, maybe let's keep this consistent?

Make sense. i have changed it.

**Pull request merged**